besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1333,8 +1334,15 @@ static class TxPoolOptionGroup {
   private Collection<EnodeURL> staticNodes;
   private BesuController besuController;
   private BesuConfiguration pluginCommonConfiguration;
+
+  private BesuComponent besuComponent;
```

Comment:
What's the difference between BesuComponent and DaggerBesuComponent?

Reply:
BesuComponent is ours, the Dagger_ prefix indicates it is the implementation that Dagger generates for us during precompile. At some point, once we've "daggerized all the things" we won't need to use these because Dagger will more fully compose the object graph.